data/is_ocaml_yet/web.md

```diff
@@ -0,0 +1,278 @@
+    packages:
+    - js_of_ocaml
+    - melange
+    - brr
+    - gen_js_api
+    - bonsai
+    - ocaml-vdom
+    - fmlib_browser
+    - tyxml
+  - name: Compression
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - zlib
+    - decompress
+    - camlzip
+    - lz4
+    - ezgzip
+    - hpack
+    - brotli
+    - zstd
+    - snappy
+    - lzo
+    - zstandard
+    - carton
+  - name: Crypto
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - cryptokit
+    - mirage-crypto
+    - nocrypto
+    - pkcs11
+    - tls
+    - x509
+    - fiat-p256
+    - sha
+    - blake2
+    - blake3
+    - bls12-381
+    - pbkdf
+    - dirsp-proscript
+    - digestif
+  - name: Database
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - caqti
+    - pgocaml
+    - sqlite3
+    - lmdb
+    - irmin
+    - mysql
+    - postgresql
+    - aws-rds
+    - sequoia
+    - sqlgg
+    - petrol
+    - mariadb
+  - name: Email
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - tidy_email
+    - emile
+    - smtp
+    - letters
+    - mrmime
+    - sendmail
+    - received
+    - email_message
+    - colombe
+    - dkim
+    - uspf
+  - name: En- & Decoding
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - tar
+    - imagelib
+    - base64
+    - multipart-form-data
+    - multipart_form
+    - biniou
+  - name: Web Frameworks
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - dream
+    - eliom
+    - opium
+    - sihl
+  - name: HTTP Clients
+    status: 🔴
+    description: |
+      n/a
+    packages:
+    - cohttp
+    - hyper
+    - ocurl
+    - http-lwt-client
+    - ezcurl
+  - name: Internationalization
+    status: 🆘
+    description: |
+      n/a
+    packages:
+    - gettext
+  - name: Logging
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - logs
+    - dolog
+    - tracing
+    - catapult
+    - opentelemetry
+    - prometheus
+  - name: Lower Web-Stack
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - httpaf
+    - http
+    - tls
+    - websocket
+    - websocketaf
+    - gluten
+    - awa
+    - uri
+    - dns
+    - tcpip
+    - charrua
+    - mirage-nat
+    - arp
+    - ethernet
+    - paf
+    - mimic
+    - tiny_httpd
+  - name: Serializers
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - atd
+    - yojson
+    - jsonm
+    - sexplib
+    - yaml
+    - csv
+    - bencode
+    - toml
+    - xml-light
+    - graphql
+    - omd
+    - cmarkit
+    - css
+    - csexp
+    - data-encoding
+  - name: External Services
+    status: 🆘
+    description: |
+      n/a
+    packages:
+    - amqp-client
+    - kafka
+    - redis
+    - zmq
+  - name: Syndication/RSS
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - syndic
+    - river
+    - rss
+  - name: Templating
```

Comment:
It should probably be mentioned here that templating is not the only option for generating HTML in OCaml and that libraries like Tyxml also exist to embed an HTML DSL directly into the language, which may be preferable for several reasons.